public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -244,20 +244,24 @@
     @Override
     public ResponseEntity<ConceptListResponse> getConceptsSearch(
             String conceptName,
-            String standardConcept,
+            String standard_concept,
             String concept_code,
```

Comment:
I'd say take out concept_code and vocabulary_id for now if they are not used anywhere.

Reply:
Okay. Done. I also removed the yaml params.